src/index.js

```diff
@@ -1,10 +1,27 @@
 /**
  * @flow
  */
-import { getThreads } from './store';
+import { getThreads, getMailboxes } from './store';
 
-function renderSidebar() {
-  const messageLIs = getThreads().reduce((html, thread) => `${html} <li>
+/* Called once at the beginning of pageload. In the function, call
+getMailboxes() to find out which ones to render based on what's in the JSON
+file. And since INBOX is the default mailbox to render, start with applying the
+"active" class to that li for styling purposes  */
+function renderNavigation() {
+  const mailboxNames = getMailboxes();
+  const navigationUL = document.querySelector('.client-nav-items');
+  const navItems = mailboxNames.map(name =>
+    name === 'INBOX' ? `<li class="client-nav-item active">${name}</li>` :
```

Comment:
Fair point. I'll rename it. It doesn't make sense to me to have it behave similarly to `renderSidebar` because that would cause it to reload/re-render on each click, which doesn't seem necessary. Thanks!